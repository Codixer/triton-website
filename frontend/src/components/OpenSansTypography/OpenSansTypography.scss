:local(.openSansTypography) {
  font-family: "Open Sans", sans-serif;
}
