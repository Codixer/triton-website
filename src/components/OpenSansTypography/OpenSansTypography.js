import React, { Component } from "react";
import styles from "./OpenSansTypography.scss";
import { Typography } from "@rmwc/typography";
import "@material/typography/dist/mdc.typography.css";

class OpenSansTypography extends Component {
  render() {
    return <Typography className={styles.openSansTypography} {...this.props} />;
  }
}

export default OpenSansTypography;
