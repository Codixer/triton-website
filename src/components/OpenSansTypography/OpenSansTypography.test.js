import React from "react";
import OpenSansTypography from "./OpenSansTypography";
import { shallow } from "enzyme";

describe("OpenSansTypography Component", () => {
  it("matches snapshot", () => {
    const wrapper = shallow(
      <OpenSansTypography use="headline1">Test Typography</OpenSansTypography>
    );
    expect(wrapper).toMatchSnapshot();
  });
});
