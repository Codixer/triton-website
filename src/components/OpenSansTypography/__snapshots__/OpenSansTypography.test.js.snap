// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OpenSansTypography Component matches snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <OpenSansTypography
    use="headline1"
  >
    Test Typography
  </OpenSansTypography>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": "Test Typography",
      "className": undefined,
      "use": "headline1",
    },
    "ref": null,
    "rendered": "Test Typography",
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": "Test Typography",
        "className": undefined,
        "use": "headline1",
      },
      "ref": null,
      "rendered": "Test Typography",
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
